WordPress/openverse
# Add the UI toggle for changing theme

## Fixes



Fixes #4309 by @zackkrida 
Fixes #4155 by @fcoveram 

## Description



This PR
- adds icon assets and i18n strings for theme switching
- updates the `VSelectField` component to hide the active choice
- creates the UI component for switching between color modes
  - wires the component to functionality implemented in #4810
  - adds the UI component to the footer based on whether the "dark_mode_ui_toggle" feature flag is enabled
- deletes the "force_dark_mode" feature flag as it is redundant
  - deletes usage and tests for the feature flag

## Testing Instructions


0. Check out the PR and run the dev server.
1. Enable the "dark_mode_ui_toggle" feature flag. You should see the theme selector in the footer.
2. Switch between light and dark theme. The site should update to show the your chosen color theme (regardless of the system theme). The icon in the footer should also update.
3. Switch between the OS light and dark theme. The text for the "System" choice in the theme selector should update.
4. Switch to the "System" theme.
5. Switch between the OS light and dark theme. The text for the "System" choice in the theme selector should update. The actual dark/light theme should also update. The icon in the footer should also update.

## Checklist


      PRs) or the media properties generator (`ov just catalog/generate-docs media-props`
      for the catalog or `ov just api/generate-docs` for the API) where applicable.

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
